libraries/AP_HAL_ChibiOS/RCOutput.cpp

```diff
@@ -2350,4 +2333,86 @@ void RCOutput::serial_led_send(const uint16_t chan)
     }
 }
 
+void RCOutput::timer_info(ExpandingString &str)
+{
+    // a header to allow for machine parsers to determine format
+    str.printf("TIMERV1\n");
+
+    for (auto &group : pwm_group_list) {
+        unsigned tim_num = 0;
+        if (false) {
+#ifdef TIM1_BASE
```

Comment:
i think we're better adding a const uint8_t timer_number in the struct pwm_group structure, and filling it in chibios_hwdef.py, it will be less code

Reply:
yep